README.rst

```diff
@@ -72,7 +72,8 @@ By default, GPU support is built if CUDA is found and ``torch.cuda.is_available(
 It's possible to force building GPU support by setting ``FORCE_CUDA=1`` environment variable,
 which is useful when building a docker image.
 
-Image Backend
+
+**Image Backend**
```

Comment:
From rendering it on markdown, there doesn't seem to be any difference by making the titles bold, as it's already done by default. Can you revert those changes?